commodore/component-template/{{ cookiecutter.slug }}/Makefile

```diff
@@ -49,6 +49,18 @@ docs-serve: ## Preview the documentation
 test: commodore_args += -f tests/$(instance).yml
 test: .compile ## Compile the component
 
+.PHONY: gen-golden
```

Comment:
Do we want to have the targets even if it is not enabled for the component? Consider to wrap them in conditions.

Reply:
I totally missed those when implementing the flags

Reply:
added in latest version